docs/source/i18n/ultimate.ar.rst

```diff
@@ -0,0 +1,23 @@
+البيانات التي نشاركها مع عامة الناس لن يظهر عليها اسمك، فقط رقم الرمز، لذلك لن يعرف الناس اسمك أو اي من البيانات التابعة لك. بالإضافة إلى ذلك، لن نشارك أي معلومات أخرى نعتقد أنها قد تساعد الأشخاص الذين يعرفونك على تخمين أي من البيانات التي تخصك.
+إذا غيرت رأيك في المستقبل و قررت سحب موافقتك على المشاركة في هذه الدراسة يمكنك الاتصال بـ (اسم المحقق الأساسي) على (رقم الهاتف) للقيام بذلك، وعندها لن نجمع أي بيانات إضافية عنك. كما اننا سنحذف بياناتك إذا قمت بالسحب قبل إيداعها في قاعدة البيانات. ولكن، لا يمكن تدمير أو سحب أو استرجاع أي بيانات ونتائج بحثية بعد أن قد تم مشاركتها مع محققين آخرين أو عامة الناس. 
+ موافقتك على المشاركة في هذه الدراسة تشكل هدية مجانية وسخية للبحث بإمكانها مساعدة الآخرين. من الممكن أن تؤدي بعض الأبحاث التي تم إجراؤها باستخدام معلوماتك في النهاية إلى تطوير طرق جديدة لدراسة الدماغ، أو اختبارات تشخيصية جديدة، أو أدوية جديدة، أو منتجات تجارية أخرى. في حالة حدوث ذلك، لا توجد ترتيبات لتزويدك بأي جزء من الأرباح الناتجة عن هذه المنتجات ولن يكون لديك أي حقوق ملكية في المنتجات.
+على حد علمنا ، لن تحتوي البيانات التي ننشرها للجمهور العام على اي معلومات يمكنها ان تؤدي الى التعرف عليك مباشرة. لن تحتوي البيانات على اسمك، فقط رقم الرمز، لذلك لن يعرف الناس اسمك أو البيانات التابعة لك. بالإضافة إلى ذلك، لن تتضمن البيانات معلومات نعتقد أنها قد تساعد الأشخاص الذين يعرفونك في تخمين البيانات التي تخصك، مثل ميزات وجهك أو التاريخ الذي شاركت فيه. إذا قمنا بكتابة تقرير أو مقال حول هذه الدراسة أو شاركنا مجموعة بيانات الدراسة مع الآخرين ، فسوف نقوم بذلك بطريقة لا يمكن التعرف عليك فيها بشكل مباشر. ومع ذلك، باستخدام بيانات إضافية مرتبطة باسمك (على سبيل المثال، تصوير دماغ شعاعي تم الحصول عليه من سجلاتك الطبية) قد يستطيع المرء أن يربط بينك وبين التصوير الخاص بك أو معلومات أخرى في قاعدة البيانات. بالإضافة إلى ذلك، قد يؤدي اختراق أمني (على شكل اختراق أو هجوم إلكتروني) إلى تميكن شخص ما من الربط بينك وبين بياناتك. هذا الخطر منخفض جدًا لأنه سيتم تخزين بياناتك في قاعدة بيانات آمنة، وسيتم تخزين المعلومات المتعلقة بهويتك بشكل منفصل عن البيانات نفسها، ويتم ربطها فقط من خلال رمز سري.
+سنحتفظ بالجزء الذي يحتوي على معلوماتك الذاتية (الاسم ومعلومات الاتصال وما إلى ذلك) من بياناتك في مكان آمن لمدة (س) سنه على الأقل.
+بهذه الطريقة يمكننا الاتصال بك إذا وجد أحد الباحثين شيئًا ما في صور دماغك الشعاعيه وكان لها قيمة تشخيصية. بعد هذه الفترة الزمنية، سوف ندمر هذه المعلومات لحماية خصوصيتك.  
+إن السماح لنا باستخدام بياناتك ومشاركتها أمر طوعي. ولكن، يجب أن تكون على استعداد لمشاركة بياناتك بهذه الطريقة من أجل المشاركة في هذه الدراسة. إذا لم تكن راغبًا بذلك، فلن يكون بامكانك المشاركة في هذه الدراسة.
+من خلال التوقيع أدناه ، فإنك توافق على تقديم بياناتك للبحث في المستقبل. أنت توافق على أنه يمكن مشاركتها مع محققين آخرين في مؤسسات أخرى من جميع أنحاء العالم. التفاصيل والنتائج والآثار المترتبة على هذه الدراسات غير معروفة
```

Comment:
I can't read it but since there is no final period, may be this sentence is (still) incomplete?

Reply:
There is no period because it would appear on the right side of the text rather than on the left. Same issue @vborghe identified. I think it is okay the way it is.

Reply:
So you'd say the current text is ready to go, @KyanY and @laraharb ? Thanks for all your work and patience!